code/frameworks/nextjs/src/font/babel/index.test.ts

```diff
@@ -22,9 +22,9 @@ const randomObj = {}
 it('should transform AST properly', () => {
```

Comment:
Could we introduce another test to test that `@next/font` imports are still working? :)

Reply:
Done!